src/main/java/frc/robot/Constants.java

```diff
@@ -277,15 +278,16 @@ public static final class constVision {
   }
 
   public static final class constLEDs {
-    public static final PatternType HAS_CONE_COLOR = PatternType.Yellow;
-    public static final PatternType HAS_CUBE_COLOR = PatternType.Violet;
+    public static final PatternType HAS_GAME_PIECE_COLOR = SN_Blinkin.PatternType.Green;
 
-    public static final PatternType DESIRED_CONE_COLOR = PatternType.StrobeGold;
-    public static final PatternType DESIRED_CUBE_COLOR = PatternType.StrobeBlue;
+    public static final PatternType DESIRED_CUBE_COLOR = SN_Blinkin.PatternType.StrobeBlue;
+    public static final PatternType DESIRED_CONE_COLOR = SN_Blinkin.PatternType.StrobeGold;
```

Comment:
I like the idea of the strobe color for a short period of time, but if it always strobes it will be really annoying. make sure it only strobes for a second or two at a time.